node/node.go

```diff
@@ -195,6 +195,7 @@ func (n *Node) Start(ctx context.Context) error {
 			return fmt.Errorf("NewClient: cannot parse private key: %w", err)
 		}
 		operator := &Operator{
+			Address:    crypto.PubkeyToAddress(privateKey.PublicKey).Hex(),
```

Comment:
What's the bug in old code? It looks more like we drop the unnecessary flag for address, since address can be derived with private key